README.md

```diff
@@ -287,7 +287,7 @@ We have confirmed that cuSignal successfully builds and runs on Windows by using
     pip install cupy-cudaXXX
     ```
 
-    Where XXX is the version of the CUDA toolkit you have installed. 11.5, for example is `cupy-cuda115`. See the [CuPy Documentation](https://docs-cupy.chainer.org/en/stable/install.html#install-cupy) for information on getting Windows wheels for other versions of CUDA.
+    Where XXX is the version of the CUDA toolkit you have installed. 11.5, for example is `cupy-cuda115`. See the [CuPy Documentation](https://docs-cupy.chainer.org/en/stable/install.html#install-cupy) for information on getting wheels for other versions of CUDA.
```

Comment:
I was searching for content that might need updated regarding CUDA 11.8 and fixed an issue here. I removed the word "Windows" because the wheels needed for WSL are _not_ Windows wheels. They're just normal Linux wheels.